NextcloudTalk/NCSettingsController.m

```diff
@@ -162,6 +165,48 @@ - (void)setActiveAccountWithAccountId:(NSString *)accountId
     [[NCConnectionController sharedInstance] checkAppState];
 }
 
+- (void)createAccountsFile
+{
+    // Create Talk directory in apps group container
+    NSString *path = [[[[NSFileManager defaultManager] containerURLForSecurityApplicationGroupIdentifier:appsGroupIdentifier] URLByAppendingPathComponent:kTalkDatabaseFolder] path];
+    if (![[NSFileManager defaultManager] fileExistsAtPath:path]) {
+        [[NSFileManager defaultManager] createDirectoryAtPath:path withIntermediateDirectories:YES attributes:nil error:nil];
+    }
+    NSURL *accountsFileURL = [[NSURL fileURLWithPath:path] URLByAppendingPathComponent:kTalkAccountsFileName];
+
+    // Create accounts data
+    NSMutableArray *accounts = [NSMutableArray new];
+    for (TalkAccount *account in [[NCDatabaseManager sharedInstance] allAccounts]) {
+        NSString *avatarPath = [self copyUserAvatarInPath:path forAccount:account];
+        NKDataAccountFile *accountFileData = [[NKDataAccountFile alloc] initWithUrl:account.server user:account.user alias:account.userDisplayName avatar:avatarPath];
+        [accounts addObject:accountFileData];
+    }
+
+    if (!accounts.count) {
```

Comment:
Not sure, this would mean that we never remove/empty a file, right? If there are no accounts, the file should be deleted?
Just tested, I removed all accounts from Talk, but still can't import all accounts to files, because the account file is not updated when removing an account and after restarting the app, this condition prevents updating/emptying as well.